lib/pxenforcer.js

```diff
@@ -73,6 +83,50 @@ class PxEnforcer {
         }
     }
 
+    isTelemetryCommand(req) {
```

Comment:
What about creating a new file called "telemetry_handler.js" and placing `isTelemetryCommand` and `sendEnforcerTelemetry` inside?

Reply:
moved it to new file